peps/pep-0730.rst

```diff
@@ -176,30 +176,27 @@ devices.
 * ``"iphonesimulator-arm64"`` for ARM64 simulators
 * ``"iphonesimulator-x86_64"`` for x86_64 simulators
 
-``sys.implementation`` will also have an additional attribute - ``_simulator`` -
-storing a Boolean that is ``True`` if the device running the app is a simulator.
-This attribute would not exist on non-iOS platforms.
-
 ``platform``
 ''''''''''''
 
-Platform will be used as the primary mechanism for retrieving OS and device
-details.
-
-* ``platform.system()`` - ``"iOS"``
+Platform be modified to support returning iOS-specific details. Most of the
+values returned by the platform module will match those returned by
+``os.uname()``, with the exception of:
 
-* ``platform.node()`` - the user-provided name of the device, as returned by the
-  ``[[UIDevice currentDevice] systemName]`` system call (e.g.,
-  ``"Janes-iPhone"``). For simulated devices, this will be the name of the
-  development computer running the simulator.
+* ``platform.system()`` - ``"iOS"``, instead of the default ``"Darwin"``
 
-* ``platform.release()`` - the iOS version number, as a string (e.g., ``"16.6.1"``)
+* ``platform.release()`` - the iOS version number, as a string (e.g.,
+  ``"16.6.1"``), instead of the Darwin kernel version.
 
-* ``platform.machine()`` - The device model returned by ``[[UIDevice
-  currentDevice] model]`` (e.g., ``"iPhone13,2"``); or ``"iPhoneSimulator"`` for
-  simulated devices.
+In addition, a ``platform.ios_ver()`` method will be added. This mirrors
+``platform.mac_ver()``, which can be used to provide macOS version information.
+``ios_ver()`` will return a namedtuple that contains the following:
 
-All other values will be as returned by ``os.uname()``
+* ``version`` - the iOS version, as a string (e.g., ``"16.6.1"``).
+* ``min_version`` - the minimum supported iOS version, as a string (e.g., ``"12.0"``)
+* ``model`` - the model identifier of the device (e.g., ``iPhone13,2``). On simulators, this
+  will return ``iPhoneSimulator``.
```

Comment:
Also a string?
```suggestion
* ``model`` - the model identifier of the device, as a string (e.g., ``iPhone13,2``).
  On simulators, this will return ``"iPhoneSimulator"``.
```